akka-sample-sharding-scala/.scalafmt.conf

```diff
@@ -0,0 +1,47 @@
+version = 2.1.0
+
+style = defaultWithAlign
+
+docstrings                 = JavaDoc
+indentOperator             = spray
+maxColumn                  = 120
+rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
+unindentTopLevelOperators  = true
+align.tokens               = [{code = "=>", owner = "Case"}]
+align.openParenDefnSite    = false
+align.openParenCallSite    = false
+optIn.breakChainOnFirstMethodDot = false
+optIn.configStyleArguments = false
+danglingParentheses = false
+spaces.inImportCurlyBraces = true
+rewrite.neverInfix.excludeFilters = [
+  and
+  min
+  max
+  until
+  to
+  by
+  eq
+  ne
+  "should.*"
+  "contain.*"
+  "must.*"
+  in
+  ignore
+  be
+  taggedAs
+  thrownBy
+  synchronized
+  have
+  when
+  size
+  only
+  noneOf
+  oneElementOf
+  noElementsOf
+  atLeastOneElementOf
+  atMostOneElementOf
+  allElementsOf
+  inOrderElementsOf
+  theSameElementsAs
+]
```

Comment:
it's unfortunate that we have to copy this into each and every sample, but I guess there is no other way

Reply:
I wondered about a few commonalities each (scala at least) sample project has and if we want to make some base plugin or other solution for common build work.

Reply:
For later https://github.com/akka/akka-samples/issues/139